src/core/src/op/util/multi_subgraph_base.cpp

```diff
@@ -160,7 +161,17 @@ void ov::op::util::MultiSubGraphOp::validate_and_infer_type_body(
         body_parameter->set_partial_shape(input_partial_shape);
         body_parameter->set_element_type(dtype);
     }
-    body->validate_nodes_and_infer_types();
+    try {
+        body->validate_nodes_and_infer_types();
+    } catch (std::exception& e) {
+        if (m_bodies.size() > 1) {
+            // If one body cannot be validated it may mean that it is not needed and will be removed on future stages
+            OPENVINO_DEBUG << "Exception happened during validation of one of the body of MultiSubGraphOp: " << e.what()
+                           << '\n';
+        } else {
+            throw e;
+        }
```

Comment:
Suspicious change because I think we won't have the situation that some body graph for TF FE will be skipped. Can we do this change only for PT FE?